projects/angular-reactive-validation/package.json

```diff
@@ -19,9 +19,9 @@
   "license": "MIT",
   "private": false,
   "peerDependencies": {
-    "@angular/common": "^17.0.0",
-    "@angular/core": "^17.0.0",
-    "@angular/forms": "^17.0.0",
+    "@angular/common": "^18.0.3",
```

Comment:
Thanks for your contribution. Could you change these to "^18.0.0" to cover all v18 versions?